components/powershaper.json

```
@@ -0,0 +1,6 @@
+{
+  "name": "PowerShaper",
+  "owner": ["@carboncoop"],
+  "manifest": "https://gitlab.com/carboncoop/hems/raw/master/homeassistant/custom_components/powershaper/manifest.json",
```

```suggestion
  "manifest": "https://gitlab.com/carboncoop/hems/-/raw/dev/homeassistant/custom_components/powershaper/manifest.json?inline=false",
```
maybe this will work :man_shrugging: , you should also rebase since the actions have changed quite a bit since the PR was opened.